internal/rbd/rbd_journal.go

```diff
@@ -533,10 +533,16 @@ func undoVolReservation(ctx context.Context, rbdVol *rbdVolume, cr *util.Credent
 // Generate new volume Handler
 // The volume handler won't remain same as its contains poolID,clusterID etc
 // which are not same across clusters.
+// nolint:gocyclo,cyclop // TODO: reduce complexity
 func RegenerateJournal(
 	volumeAttributes map[string]string,
-	volumeID, requestName, owner string,
-	cr *util.Credentials) (string, error) {
+	claimName,
+	claimNamespace,
+	volumeID,
+	requestName,
+	owner string,
```

Comment:
```suggestion
	volumeID,
	requestName,
	owner string,
```
owner refers to the namespace.